notebooks/official/custom/get_started_vertex_training_xgboost.ipynb

```diff
@@ -0,0 +1,992 @@
+{
```

Comment:
<p>Do we maybe want to avoid using the Iris dataset?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1665/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>this is a migrated notebook. No policy requiring to replace the Iris dataset in existing notebooks</p>